Configurations/unix-Makefile.tmpl

```diff
@@ -679,9 +679,12 @@ tags TAGS: FORCE
 
 # Release targets (note: only available on Unix) #####################
 
+# If your tar command doesn't support --owner and --group, make sure to
+# use one that does, for example GNU tar
 TAR_COMMAND=$(TAR) $(TARFLAGS) --owner 0 --group 0 -cvf -
```

Comment:
True.  However, we have enforced user 0 and group 0 since the start of OpenSSL (a long time ago, we used `tardy` to do that)...  I'm not sure now is the time to stop.